otter/convergence/service.py

```diff
@@ -310,11 +333,24 @@ def converge_one_group(log, group_locks, tenant_id, group_id, version,
     """
     log = log.bind(tenant_id=tenant_id, group_id=group_id)
     eff = execute_convergence(tenant_id, group_id, log)
-    return group_locks(group_id, eff).on(
-        success=lambda r: delete_divergent_flag(log, tenant_id,
-                                                group_id, version),
-        error=partial(_cleanup_convergence_error, log, tenant_id,
-                      group_id, version))
+    try:
+        result = yield group_locks(group_id, eff)
+    except ConcurrentError:
```

Comment:
This is supposed to be raised when the current Otter process is already converging, right?